### PR TITLE
🍰 2019/kw24/fix_imported_img_urls

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Fixed URLs on import to make them relative.
This works only for `api-alpha` URLs.

### Issues

- relates #692 

### Todo

- [X] None

### CODE REVIEW

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql [hunk 1]
```diff
@@ -45,7 +45,7 @@ MERGE(b:Badge {id: badge._id["$oid"]})
 ON CREATE SET
 b.key       = badge.key,
 b.type      = badge.type,
-b.icon      = badge.image.path,
+b.icon      = substring(badge.image.path, 38),
```
Comment (roschaefer) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql]: @ulfgebhardt I find it quite dangerous, how do you know that it's always the same first part of the URL?

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql [hunk 2]
```diff
@@ -28,7 +28,7 @@
 [?]     unique: true, // Unique value is not enforced in Nitro?
 [-]     index: true
       },
-[ ]   type: {
+[ ]   type: { // db.getCollection('contributions').distinct('type') -> 'DELETED', 'cando', 'post'
```
Comment (roschaefer) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql]: Why is `DELETED` a type :thinking: ?

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql (hunk 1 above)
Comment (roschaefer) [reply deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql]: Use [replace](https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-replace).

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql (hunk 2 above)
Comment (ulfgebhardt) [reply deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions.cql]: I cannot tell you - its just sure it is there

File: webapp/package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
       "!**/?(*.)+(spec|test).js?(x)"
     ],
     "coverageReporters": [
+      "text",
```
Comment (roschaefer) [inline webapp/package.json]: Unrelated?
